src/devices/Card/Mifare/MifareCard.cs

```diff
@@ -29,24 +25,24 @@ public class MifareCard
         /// <summary>
         /// Default Key A
         /// </summary>
-        public byte[] DefaultKeyA { get; } = new byte[6] { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+        public static byte[] DefaultKeyA { get; } = new byte[6] { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
```

Comment:
consider making this private and exposing it through ReadOnlySpan, as is it's writable